From baa85a4dab2e8d64eb25d1181b6420db25ce919a Mon Sep 17 00:00:00 2001
From: Trevor Gamblin <trevor.gamblin@windriver.com>
Date: Tue, 22 Jun 2021 12:31:46 -0400
Subject: [PATCH] change shebang to python3

Upstream-Status: Inappropriate (OE-specific)

Despite no longer supporting python2, some files in the pip source refer
to "python" instead of "python3", so patch them as needed to ensure that
they correctly reference the python3 binary.

Signed-off-by: Trevor Gamblin <trevor.gamblin@windriver.com>
---
 src/pip/_vendor/appdirs.py                    | 2 +-
 src/pip/_vendor/chardet/langbulgarianmodel.py | 2 +-
 src/pip/_vendor/chardet/langgreekmodel.py     | 2 +-
 src/pip/_vendor/chardet/langhebrewmodel.py    | 2 +-
 src/pip/_vendor/chardet/langhungarianmodel.py | 2 +-
 src/pip/_vendor/chardet/langrussianmodel.py   | 2 +-
 src/pip/_vendor/chardet/langthaimodel.py      | 2 +-
 src/pip/_vendor/chardet/langturkishmodel.py   | 2 +-
 src/pip/_vendor/chardet/metadata/languages.py | 2 +-
 src/pip/_vendor/requests/certs.py             | 2 +-
 10 files changed, 10 insertions(+), 10 deletions(-)

diff --git a/src/pip/_vendor/appdirs.py b/src/pip/_vendor/appdirs.py
index 33a3b7741..60b9ef5f7 100644
--- a/src/pip/_vendor/appdirs.py
+++ b/src/pip/_vendor/appdirs.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 # Copyright (c) 2005-2010 ActiveState Software Inc.
 # Copyright (c) 2013 Eddy Petri»ôor
diff --git a/src/pip/_vendor/chardet/langbulgarianmodel.py b/src/pip/_vendor/chardet/langbulgarianmodel.py
index e963a5097..97ea6cea8 100644
--- a/src/pip/_vendor/chardet/langbulgarianmodel.py
+++ b/src/pip/_vendor/chardet/langbulgarianmodel.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 
 from pip._vendor.chardet.sbcharsetprober import SingleByteCharSetModel
diff --git a/src/pip/_vendor/chardet/langgreekmodel.py b/src/pip/_vendor/chardet/langgreekmodel.py
index d99528ede..4a127ea83 100644
--- a/src/pip/_vendor/chardet/langgreekmodel.py
+++ b/src/pip/_vendor/chardet/langgreekmodel.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 
 from pip._vendor.chardet.sbcharsetprober import SingleByteCharSetModel
diff --git a/src/pip/_vendor/chardet/langhebrewmodel.py b/src/pip/_vendor/chardet/langhebrewmodel.py
index 484c652a4..676c1a711 100644
--- a/src/pip/_vendor/chardet/langhebrewmodel.py
+++ b/src/pip/_vendor/chardet/langhebrewmodel.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 
 from pip._vendor.chardet.sbcharsetprober import SingleByteCharSetModel
diff --git a/src/pip/_vendor/chardet/langhungarianmodel.py b/src/pip/_vendor/chardet/langhungarianmodel.py
index bbc5cda64..042eae70a 100644
--- a/src/pip/_vendor/chardet/langhungarianmodel.py
+++ b/src/pip/_vendor/chardet/langhungarianmodel.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 
 from pip._vendor.chardet.sbcharsetprober import SingleByteCharSetModel
diff --git a/src/pip/_vendor/chardet/langrussianmodel.py b/src/pip/_vendor/chardet/langrussianmodel.py
index 5594452b5..564b02e58 100644
--- a/src/pip/_vendor/chardet/langrussianmodel.py
+++ b/src/pip/_vendor/chardet/langrussianmodel.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 
 from pip._vendor.chardet.sbcharsetprober import SingleByteCharSetModel
diff --git a/src/pip/_vendor/chardet/langthaimodel.py b/src/pip/_vendor/chardet/langthaimodel.py
index 9a37db573..c97487959 100644
--- a/src/pip/_vendor/chardet/langthaimodel.py
+++ b/src/pip/_vendor/chardet/langthaimodel.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 
 from pip._vendor.chardet.sbcharsetprober import SingleByteCharSetModel
diff --git a/src/pip/_vendor/chardet/langturkishmodel.py b/src/pip/_vendor/chardet/langturkishmodel.py
index 43f4230ae..7e710c371 100644
--- a/src/pip/_vendor/chardet/langturkishmodel.py
+++ b/src/pip/_vendor/chardet/langturkishmodel.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 
 from pip._vendor.chardet.sbcharsetprober import SingleByteCharSetModel
diff --git a/src/pip/_vendor/chardet/metadata/languages.py b/src/pip/_vendor/chardet/metadata/languages.py
index 3237d5abf..aa2ec7c35 100644
--- a/src/pip/_vendor/chardet/metadata/languages.py
+++ b/src/pip/_vendor/chardet/metadata/languages.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 """
 Metadata about languages used by our model training code for our
diff --git a/src/pip/_vendor/requests/certs.py b/src/pip/_vendor/requests/certs.py
index 06a594e58..bfa7839a4 100644
--- a/src/pip/_vendor/requests/certs.py
+++ b/src/pip/_vendor/requests/certs.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 
 """
-- 
2.31.1

